src/Symfony/Component/Form/Tests/Extension/Validator/ValidatorTypeGuesserTest.php

```diff
@@ -19,6 +19,7 @@
 use Symfony\Component\Validator\Constraints\NotBlank;
 use Symfony\Component\Validator\Constraints\NotNull;
 use Symfony\Component\Validator\Constraints\Range;
+use Symfony\Component\Validator\Constraints\IsTrue;
 use Symfony\Component\Validator\Constraints\True;
```

Comment:
can you try with `use ..\True as LegacyTrue;` or is there another way to avoid Fatal error ?
`PHP Fatal error:  Cannot use Symfony\Component\Validator\Constraints\True as True because 'True' is a special class name in `